versioned_docs/version-v2.9.0/installation/platforms/aks.md

```
@@ -0,0 +1,201 @@
+## Step 4: Verify Your Installation
+
+### 1. Verify Pod Status
+
+Check that the HAMi scheduler and device plugin pods are in `Running` state:
+
+```bash
+kubectl get pods -n kube-system -l app.kubernetes.io/name=hami
+```
+
+Expected output:
+```text
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)